packages/lsp/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@jupyterlab/lsp",
+  "version": "4.0.0-alpha.11",
+  "description": "",
+  "homepage": "https://github.com/jupyterlab/jupyterlab",
+  "bugs": {
+    "url": "https://github.com/jupyterlab/jupyterlab/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jupyterlab/jupyterlab.git"
+  },
+  "license": "BSD-3-Clause",
+  "author": "Project Jupyter",
+  "sideEffects": [
+    "style/*.css",
+    "style/index.js"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "directories": {
+    "lib": "lib/"
+  },
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "schema/*.json",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
+    "style/index.js"
+  ],
+  "scripts": {
+    "build": "jlpm build:schema && tsc -b",
+    "build:all": "npm run build",
+    "build:schema": "node ./schema.js",
+    "build:test": "tsc --build tsconfig.test.json",
+    "clean": "rimraf lib && rimraf tsconfig.tsbuildinfo",
+    "test": "jest",
+    "test:cov": "jest --collect-coverage",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
+    "test:debug:watch": "node --inspect-brk node_modules/.bin/jest --runInBand --watch",
+    "watch": "tsc -b --watch"
+  },
+  "resolutions": {
+    "vscode-languageserver-protocol": "^3.17.0"
+  },
+  "dependencies": {
+    "@jupyterlab/apputils": "^4.0.0-alpha.11",
+    "@jupyterlab/codeeditor": "^4.0.0-alpha.11",
+    "@jupyterlab/coreutils": "^6.0.0-alpha.11",
+    "@jupyterlab/docregistry": "^4.0.0-alpha.11",
+    "@jupyterlab/services": "^7.0.0-alpha.11",
+    "@jupyterlab/translation": "^4.0.0-alpha.11",
+    "@lumino/coreutils": "^1.12.0",
+    "@lumino/disposable": "^1.10.1",
+    "@lumino/signaling": "^1.10.1",
+    "codemirror": "~5.61.0",
+    "lodash.mergewith": "^4.6.1",
+    "lsp-ws-connection": "~0.7.0",
```

Comment:
I think we should remove this dependency, it was getting axed anyways (I think this is incorrectly ticked as done in the PR description). It only provides `registerServerCapability` and `unregisterServerCapability` and `LspWsConnection`; let's merge [`LspWsConnection`](https://github.com/jupyter-lsp/jupyterlab-lsp/blob/240f2cbda77f574ab98ae65dafea8b694ecb13f5/packages/lsp-ws-connection/src/ws-connection.ts#L30-L33) into `LSPConnection` removing `@deprecated` methods (everything which uses `events.EventEmitter` or old `get`/`send` API).